keps/sig-apps/1847-autoremove-statefulset-pvcs/kep.yaml

```diff
@@ -25,7 +25,7 @@ latest-milestone: "v1.27"
 milestone:
```

Comment:
nit: we can update latest-milestone as well

Reply:
Sorry, missed this.

Updated to 1.31, is that what we mean by latest-milestone?